packages/clerk-js/src/ui/components/Checkout/Checkout.tsx

```diff
@@ -36,9 +36,7 @@ const AuthenticatedRoutes = withCoreUserGuard((props: __experimental_CheckoutPro
       <Drawer.Overlay />
       <Drawer.Content>
         <Drawer.Header title='Checkout' />
-        <Drawer.Body>
-          <CheckoutPage {...props} />
-        </Drawer.Body>
+        <CheckoutPage {...props} />
```

Comment:
Isn't that tying the checkout page to be always in a drawer? Do we want users to be able to embed this onto a separate page?

Reply:
`<Checkout />` would be what the user implements on the page which always renders in the `<Drawer />`.